app/Domain/Notifications/KeyHasBeenGivenNotification.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Toby\Domain\Notifications;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Notification;
+use Toby\Eloquent\Models\User;
+
+class KeyHasBeenGivenNotification extends Notification
+{
+    use Queueable;
+
+    public function __construct(
+        protected User $sender,
+        protected User $recipient,
+    ) {}
+
+    public function via(): array
+    {
+        return ["slack"];
```

Comment:
Maybe these `"slack"` and other channels could be gathered somewhere and used as constants or enums?